### ISSUE TITLE
Support for 4-byte UTF-8 characters

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/purl-fetcher#729]
Purl-fetcher currently rejects https://argo.stanford.edu/view/zt482bf4435 becuase it uses a character that would cause the database to fail because it does not support UTF-8 4byte.  

The character is https://codepoints.net/U+2B78E?lang=en 

> CJK Unified Ideographs Extension D is a Unicode block containing rare and historic CJK ideographs for Chinese, Japanese, Korean, and Vietnamese.

This may get fixed by doing a migration to PostgreSQL.

**Comment by thatbudakguy** [comment]
I have no idea if this is acceptable practice, but removing the text of the poem (青い空　赫い𫞎骨　満々と流るゝ水に　近代文化の笑顔が覗いてゐる) from the title would solve the issue.

**Comment by cbeer** [comment]
I'm not sure I understand why we're rejecting the whole record. In https://github.com/sul-dlss/purl-fetcher/pull/606, I thought we agreed to just drop the problematic title?

**Comment by jcoyne** [comment]
Yeah, it looks like we have an outright rejection in PurlsController#update
